base/build.h

```diff
@@ -1,23 +0,0 @@
-// This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
I think strictly speaking no, just this file deletion could have been committed to master by itself at any time.